### PR TITLE
docs: improve Html documentation

### LINKED ISSUE #116: How to use 'colspan' and 'rowspan' in a Html components ?
I try to create a `v.SimpleTable`I would like to create a first line with a colspan > 1
I try the following but it didn't end up as expected : 
```py
import ipyvuetify as v

titles = [f'range {i}' for i in range(4)]

description = v.Html( tag = 'tr', children = [
    v.Html(tag='th', children = ['']),
    v.Html(tag = 'th', colspan = str(len(titles)), children = ['Un gros titre'])
])


# create a header 
header = v.Html(tag = 'tr', children = (
    [v.Html(tag = 'th', children = [''])] 
    + [v.Html(tag = 'th', children = [class_]) for class_ in titles]
))

# create a row
rows = []
for i, baseline in enumerate(titles):
            
    inputs = []
    for j, target in enumerate(titles):
                
        input_ = v.Html(tag='td', class_='ma-0 pa-0', children=['niet'])
        inputs.append(input_)
                
    row = v.Html(tag='tr', children=(
        # [v.Html(tag='td', class_='ma-0 pa-0', children=[''])]
        [v.Html(tag='th', children=[baseline])] 
        + inputs
    ))
             
    rows.append(row)

#assemble the table
table =  v.SimpleTable(children = [
    v.Html(tag = 'thead', children = [description]),
    v.Html(tag = 'tbody', children = [header] + rows)
])
```

Capture d’écran 2020-12-22 à 21 52 52

Is it possible to use these keywords without using a vuetifytemplate ?

### PR DESCRIPTION
**Pull request by 12rambau**
add the attributes trait in the description on the Html component
Fix #116

### CODE REVIEW

**Comment by mariobuikhuizen** [comment]
Thanks for this contribution!

I don't think the first commit should be here. You might want to rebase from upstream/master.

**Comment by 12rambau** [comment]
Never rebased a repository before, I'll update the branch when I'll understand how it works

File: docs/usage.rst [hunk 1]
```diff
@@ -253,21 +253,26 @@ Regular HTML tags
...
 
 ipyvuetify
 
 .. jupyter-execute::
 
     v.Container(children=[
-        v.Html(tag='h1', children=['My heading'])
+        v.Html(
+            tag='h1', 
+            attributes={'title': "I'm a title"},
```
Comment (mariobuikhuizen) [inline docs/usage.rst:273]: We try to be consistent in using single quotes for python strings.

Comment (mariobuikhuizen) [reply docs/usage.rst]: I'm sorry, I overlooked the single quote in the string. Makes sense to use double quotes then. (no need to change your solution to this)

**Pull request merged**